app/elements/coder/team/create-team-modal.html

```diff
@@ -60,32 +60,24 @@
 			listeners: {
 				'keyup': '_teamChanged'
 			},
-			toggle: function(){
-				this.$.createTeamDialog.toggle();
+			open: function(){
+				this.$.createTeamDialog.open();
 			},
 			_saveTeam: function(){
 				if(!this.$.form.validate()){
 					return;
 				}
-				this.$.createTeamRequest.body = this.team;
-				this.$.createTeamRequest.url = util.build('/teams');
 				this.$.createTeamRequest.generateRequest();
 				this.team = {};
```

Comment:
make this empty when u get a response. If it is failed then maybe the user wants to modify what he wrote